bonsai/sdk/src/alpha.rs

```diff
@@ -95,6 +95,23 @@ pub mod responses {
         ///
         /// If the status == 'SUCCEEDED' then this should be present
         pub receipt_url: Option<String>,
+        /// Session Error message
+        ///
+        /// If the session is not RUNNING or SUCCEEDED, this is the error raised from within bonsai.
+        pub error_msg: Option<String>,
+        /// Session Proving State
+        ///
+        /// Indication of where in the proving pipeline the session currently is.
+        /// Possible states in order, include:
```

Comment:
Perhaps indicate that this is `None` if the `status` isn't `RUNNING` (or does it also list state at exit in the case of `FAILED` / `TIMED_OUT` / `ABORTED`?)

Reply:
Also this makes me notice that `SUCCEEDED` is double-listed on this line (and the line preceding it): https://github.com/risc0/risc0/pull/843/files#diff-9764b4c0c61823859f73ff49097c3ae82c7e8d9256e748ddaef9c66030e22df7R92

Reply:
Thanks @tzerrell ! Good notes, I took a stab at refining it here: c63ebcf8a835a02733d6620a80987e9c434bc323 

Attached is a screenshot of what the formatting looks like. Thoughts? 
![2023-09-07_12-09-1694113786](https://github.com/risc0/risc0/assets/2153856/8c9ca8a5-31f0-44aa-bddd-c0e73c560359)